nutrinova-api/NutrinovaData/NutrinovaDbContext.cs

```diff
@@ -49,72 +51,75 @@ public NutrinovaDbContext(DbContextOptions<NutrinovaDbContext> options)
     public virtual DbSet<Unit> Units { get; set; }
 
     protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
-
-        // #warning To protect potentially sensitive information in your connection string, you should move it out of source code. You can avoid scaffolding the connection string by using the Name= syntax to read it from configuration - see https://go.microsoft.com/fwlink/?linkid=2131148. For more guidance on storing connection strings, see http://go.microsoft.com/fwlink/?LinkId=723263.
         => optionsBuilder.UseNpgsql("Host=nutrinova-db;Port=5432;Database=nutrinovadb;Username=admin;Password=Pleasegivemecoke!");

```

Comment:
Should we pull these out into a secret?

Reply:
eventually, right now its just for our development db